cmd/crc/cmd/status.go

```diff
@@ -0,0 +1,73 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/docker/go-units"
+	"github.com/spf13/cobra"
+	"os"
+	"path/filepath"
+	"text/template"
+
+	"github.com/code-ready/crc/pkg/crc/constants"
+	"github.com/code-ready/crc/pkg/crc/errors"
+	"github.com/code-ready/crc/pkg/crc/machine"
+)
+
+func init() {
+	rootCmd.AddCommand(statusCmd)
+}
+
+var statusCmd = &cobra.Command{
+	Use:   "status",
+	Short: "Display status of the cluster",
+	Long:  "Show details about OpenShift cluster and crc vm",
+	Run: func(cmd *cobra.Command, args []string) {
+		runStatus()
+	},
+}
+
+var statusFormat = `CRC VM:     {{.CrcStatus}}
+OpenShift:  {{.OpenShiftStatus}}
+DiskUsage:  {{.DiskUsage}}
```

Comment:
Make it explicit that this is the Disk Usage in the VM? Is this going to be printed on stdout? If yes, this should contain spaces ("Disk Usage", "Cache Usage")

Reply:
Yes, this is printed to stdout, the output looks like the following, which are separated by new lines. So we don't need the commas.
```
$ crc status                          
CRC VM:     Running
OpenShift:  Stopped
DiskUsage:  31% of 30G (Mounted On: /)
CacheUsage: 9.835GB (used by oc binary, Disk image)
```
Will mention that the disk usage is in the VM.